Source/BSN.Commons/Infrastructure/Kafka/KafkaProducerOptions.cs

```diff
@@ -3,18 +3,7 @@
     /// <inheritdoc />
     public class KafkaProducerOptions : IKafkaProducerOptions
     {
-        /// <param name="bootstrapServers"><see cref="BootstrapServers"/></param>
-        /// <param name="receiveMessageMaxBytes"><see cref="ReceiveMessageMaxBytes"/></param>
-        public KafkaProducerOptions(string bootstrapServers, string receiveMessageMaxBytes)
-        {
-            BootstrapServers = bootstrapServers;
-            ReceiveMessageMaxBytes = receiveMessageMaxBytes;
-        }
-
         /// <inheritdoc />
-        public string BootstrapServers { get; }
-
-        /// <inheritdoc />
-        public string ReceiveMessageMaxBytes { get; }
+        public string BootstrapServers { get; set; }
```

Comment:
Please use getter only property

Options must be immutable

Reply:
done